pontoon/machinery/utils.py

```diff
@@ -56,6 +59,29 @@ def get_google_translate_data(text, locale_code):
         }
 
 
+def get_concordance_search_results(text, locale):
+    search_phrases = base.utils.get_search_phrases(text)
+    search_filters = (
+        Q(Q(target__icontains_collate=(phrase, locale.db_collation)), locale=locale,)
+        for phrase in search_phrases
+    )
+    search_query = reduce(operator.and_, search_filters)
+
+    search_query_results = base.models.TranslationMemoryEntry.objects.filter(
+        search_query
+    ).values_list("target", "project__name")
+
+    search_results = [
```

Comment:
@abowler2 Hey!
Could you take a look and check the fields that are going to be returned in concordance search results? They defined are below.
Ping me on chat if you have any questions :-) Thanks!

Reply:
This looks good to me.